lib/modules/manager/nix/__fixtures__/flake.5.lock

```diff
@@ -0,0 +1,27 @@
+{
+  "nodes": {
+    "ijq": {
+      "flake": false,
+      "locked": {
+        "lastModified": 1723569650,
+        "narHash": "sha256-Ho/sAhEUeSug52JALgjrKVUPCBe8+PovbJj/lniKxp8=",
+        "owner": "~gpanders",
+        "repo": "ijq",
+        "rev": "88f0d9ae98942bf49cba302c42b2a0f6e05f9b58",
+        "type": "sourcehut"
+      },
+      "original": {
+        "owner": "~gpanders",
+        "repo": "ijq",
+        "type": "sourcehut"
```

Comment:
Could we in theory also support "type": "tarball"?

Reply:
Would be acceptable if this is too hard to do in this PR.

Reply:
see https://github.com/NixOS/nix/blob/master/doc/manual/src/protocols/tarball-fetcher.md#gitea-and-forgejo-support 

For the format. We can probably easily regex that and if it doesn't match just skip it.

Reply:
Just pushed a commit with that as well